File: app/src/main/java/taxi/kassa/model/responses/ResponseOwner.java
package taxi.kassa.model.responses;

public class ResponseOwner {

    String balance_city;
    String balance_gett;
    String balance_yandex;
    String balance_total;
    String balance_fuel;
    String first_name;
    String last_name;
    String phone;

    public String getBalanceCity() {
        return balance_city;
    }

    public String getBalanceGett() {
        return balance_gett;
    }

    public String getBalanceYandex() {
        return balance_yandex;
    }

    public String getBalanceTotal() {
        return balance_total;
    }

    public String getBalanceFuel() {
        return balance_fuel;
    }

    public String getFullName() {
        return last_name + " " + first_name;
    }

    public String getPhone() {
        return phone;
    }
}

File: app/src/main/java/taxi/kassa/model/responses/ResponseOwner.kt
package taxi.kassa.model.responses

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ResponseOwner(
    @SerializedName("balance_city")
    @Expose
    var balanceCity: String,
    @SerializedName("balance_gett")
    @Expose
    var balanceGett: String,
    @SerializedName("balance_yandex")
    @Expose
    var balanceYandex: String,
    @SerializedName("balance_total")
    @Expose
    var balanceTotal: String,
    @SerializedName("balance_fuel")
    @Expose
    var balanceFuel: String,
    @SerializedName("first_name")
    @Expose
    var firstName: String,
    @SerializedName("last_name")
    @Expose
    var lastName: String,
    @SerializedName("phone")
    @Expose
    var phone: String
) {

    val fullName: String
        get() = "$lastName $firstName"
}